include/qp_solver_collection/QpSolverCollection.h

```diff
@@ -10,7 +10,17 @@
 
 #include <qp_solver_collection/QpSolverOptions.h>
 
-#include <ros/console.h>
+#if QP_SOLVER_COLLECTION_STANDALONE
+#  include <iostream>
+#  define QSC_ERROR_STREAM(x) std::cerr << x
```

Comment:
I've not tried but don't you need `<< std::endl` because ROS_INFO_STREAM automatically breaks lines?